src/ambianic/pipeline/store.py

```diff
@@ -171,18 +175,52 @@ def process_sample(self, **sample) -> Iterable[dict]:
                 yield processed_sample
 
     def notify(self, save_json: dict):
-        if self.notification is None:
-            return
-        log.debug("Sending notification(s)..")
         # TODO extract inference data
+        if save_json['inference_result'] is None:
+            return
+
         for inference_result in save_json['inference_result']:
             data = {
                 'id': save_json['id'],
                 'label': inference_result['label'],
                 'confidence': inference_result['confidence'],
                 'datetime': save_json['datetime'],
             }
-            notification = Notification(data=data, providers=self.notification_config["providers"])
+
+            try:
+                premiumFile = pkg_resources.resource_filename(
+                    "ambianic.webapp", "premium.yaml")
+                with open(premiumFile, "r") as file:
+                    configFile = yaml.safe_load(file)
+                    userId = configFile['credentials']["USER_AUTH0_ID"]
+                    endpoint = configFile['credentials']["NOTIFICATION_ENDPOINT"]
```

Comment:
@vickywane I can't find the code or configuration that calls out the notification API endpoint.
I was expecting to see a place where the pipeline config is updated to send notifications via the premium API. How else would apprise know the credentials and endpoint where to direct event notifications.

Reply:
The configuration is stored in a `premium.yaml` file, which is created when the PWA syncs with the edge device. 

As shown in the snippet above, the credentials are gotten from the PWA through the `premium-notifcation` endpoint [here](https://github.com/vickywane/ambianic-edge/blob/auth/auth0-pwa/src/ambianic/webapp/flaskr.py#L165).
[See](https://github.com/ambianic/ambianic-ui/pull/562#discussion_r628458493) my response to this question on the PWA pull request.

You can also refer to the design document for more clarification as this process is documented there.